app/charts/map/map-legend.tsx

```diff
@@ -705,7 +705,7 @@ const ContinuousColorLegend = ({
           width={width - MARGIN.left - MARGIN.right}
           height={COLOR_RAMP_HEIGHT}
           colorInterpolator={getColorInterpolator(palette as any)}
-          nbClass={width - MARGIN.left - MARGIN.right}
+          nSteps={width - MARGIN.left - MARGIN.right}
```

Comment:
Cannot see any steps at https://visualization-tool-git-feat-most-recent-x-axis-ixt1.vercel.app/storybook/index.html?path=/story/components-colorramp--discrete, is this because nSteps is not changed there ?

Reply:
Looks like it, I updated the story to infer the types based on the component in https://github.com/visualize-admin/visualization-tool/pull/1436/commits/bec64dcf70c3a3b599b198b4005c8ee90811fa10 👍